components/Header.js

```diff
@@ -30,12 +30,13 @@ const Header = () => (
         </div>
       </div>
       <div className="mt-6 text-center">
-        <a href="#apply" className="text-base md:text-xl font-bold outline-none text-white py-3 px-6 rounded-md bg-blue-700 hover:opacity-75">
+        <a  /*href="#apply"*/  className="text-base md:text-xl font-bold outline-none text-white py-3 px-6 rounded-md bg-blue-700 hover:opacity-75"
+        className="text-base md:text-xl m-2 font-bold outline-none text-white py-3 px-6 rounded-md bg-blue-700 /*hover:opacity-75*/ opacity-50 cursor-not-allowed">
```

Comment:
`href` looks great!

Can we only have one `className` though? (The last one will overwrite the first one, but still, confusing to have two I think).

Also, looks like there are a few differences between the new and old `className` -- can we use what was there originally, just commenting out `hover:opacity-75` and adding `opacity-50 cursor-not-allowed`?

Reply:
Done.